src/lib/auth.ts

```diff
@@ -94,6 +94,9 @@ export const ROLES: {
 		'resin.service_install.all',
 		'resin.service_label.all',
 		'resin.user.read',
+
+		`resin.service_instance.get?service_type eq 'vpn'`,
+		`resin.device.tunnel-22222?`,
```

Comment:
Stray question mark right there

Reply:
You also don’t need the backticks — single or double (above) quotes will do as you’re doing no interpolation.

Reply:
One more thing :) Can you please either sort them a/b or add a nice comment about them just above so they’re grouped together?

Reply:
Fixed.